src/test/java/teammates/test/cases/ui/browsertests/AdminEmailPageUiTest.java

```diff
@@ -66,22 +79,169 @@ private void testCompose() {
         emailPage.inputSubject("Email Subject");
         emailPage.inputContent("Email to save");
         emailPage.clickSaveButton();
-        emailPage.verifyHtmlMainContent("/adminEmailSaveSuccess.html");
+        assertTrue(hasStatusMessageSaveSuccess());
     }
 
     private void testSent() {
         emailPage.clickSentTab();
-        emailPage.verifyHtmlMainContent("/adminEmailSentPage.html");
+        assertTrue(isEmailSentDataDisplayCorrect());
     }
 
     private void testDraft() {
         emailPage.clickDraftTab();
-        emailPage.verifyHtmlMainContent("/adminEmailDraftPage.html");
+        assertTrue(isEmailDraftDataDisplayCorrect());
     }
 
     private void testTrash() {
         emailPage.clickTrashTab();
-        emailPage.verifyHtmlMainContent("/adminEmailTrashPage.html");
+        assertTrue(isEmailTrashDataDisplayCorrect());
+    }
+    
+    private boolean isEmailComposePageCorrect() {
+        return emailPage.isElementPresent(By.id("addressReceiverEmails"))
+            && emailPage.isElementPresent(By.name("emailsubject"))
+            && emailPage.isElementPresent(By.className("mce-tinymce"))
+            && emailPage.isElementPresent(By.id("adminEmailBox"))
+            && emailPage.isElementPresent(By.id("composeSubmitButton"))
+            && emailPage.isElementPresent(By.id("composeSaveButton"));
```

Comment:
Should references like these be on `AdminEmailPage.java` instead?
The method name can also be something more specific like `isEmailComposeElementsPresent()`.
`is...Correct()` is honestly a pretty vague name in general.